client/state/terms/schema.js

```diff
@@ -29,3 +29,23 @@ export const termsSchema = {
 	},
 	additionalProperties: false
 };
+
+export const queriesSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			patternProperties: {
+				// Queries are JSON strings, prepended by a site ID and taxonomy slug
+				'^([A-Za-z0-9-_]+:)\\{[^\\}]*\\}$': {
```

Comment:
Ah, I see now, we're still including slug in the serialized query. What do you think about nesting this similar to what we did with sites?